index.js

```diff
@@ -3,19 +3,21 @@ var gutil = require('gulp-util');
 var es = require('event-stream');
 var amdextract = require('amdextract');
 var PluginError = gutil.PluginError;
+var isUndefined = require('lodash.isundefined');
 
 const PLUGIN_NAME = 'gulp-amdcheck';
 
 function gulpAmdCheck(options) {
   options = options || {};
 
-  options.logFilePath = options.logFilePath || true;
-  options.logModuleId = options.logModuleId || false;
-  options.logDependencyPaths = options.logDependencyPaths || false;
-  options.logDependencyNames = options.logDependencyNames || false;
-  options.logUnusedDependencyPaths = options.logUnusedDependencyPaths || true;
-  options.logUnusedDependencyNames = options.logUnusedDependencyNames || false;
-  options.removeUnusedDependencies = options.removeUnusedDependencies || true;
+  options.logFilePath = isUndefined(options.logFilePath) ? true : options.logFilePath;

```

Comment:
I agreed, its a bug. But I think `isUndefined` from lodash is not required. You can implement it:
```js
var undefined;

function isUndefined(value) {
  return value === undefined;
}
```